src/replaceRuleSelector.js

```diff
@@ -4,6 +4,21 @@ import balancedMatch from "balanced-match"
 
 const pseudoClass = ":matches"
 
+function isElementSelector(selector) {
+  return !selector.match(/(\:|\.)/g)
```

Comment:
Any feedback about this Regex? If you have any dot or colon then you are not an element.